sql/src/main/java/org/apache/druid/sql/calcite/planner/SqlResourceCollectorShuttle.java

```diff
@@ -87,13 +88,21 @@ public SqlNode visit(SqlIdentifier id)
         if (qualifiedNameParts.size() == 2) {
           final String schema = qualifiedNameParts.get(0);
           final String resourceName = qualifiedNameParts.get(1);
+
+          // Put the lookup names in the query context to facilitate selective loading of lookups.
+          if (schema.equals(NamedLookupSchema.NAME)) {
+            plannerContext.queryContextMap().putIfAbsent(PlannerContext.CTX_LOOKUPS_TO_LOAD, new HashSet<>());
```

Comment:
Since we are never initializing this value, would that allow a user to set CTX_LOOKUPS_TO_LOAD, and those would get added to the list and loaded?

Reply:
@adarshsanjeev Yes, I tried editing context from the UI with a list of lookups, and could see them being loaded.

Reply:
@cryptoe Can you please advise here? Thanks!

Reply:
@LakshSingla This should be a non-issue now, please check: https://github.com/apache/druid/pull/16358#issuecomment-2092822866

Since we've moved away from query context, users cannot override the values that get populated for MSQ tasks.